scripts/opera-rtc/query_cmr.py

```diff
@@ -0,0 +1,40 @@
+import datetime
+import json
+
+import requests
+
+
+dates = [
+    datetime.datetime(2022, 2, 1),
+    datetime.datetime(2022, 8, 1),
+    datetime.datetime(2023, 2, 1),
+    datetime.datetime(2023, 8, 1),
+    datetime.datetime(2024, 2, 1),
+    datetime.datetime(2024, 8, 1),
```

Comment:
@forrestfwilliams Can you double-check the date range for me here?

Notably, there is no S1B data available for these dates, as it died Dec 2021.